packages/web/app/api/gql/typeDefs/vault.ts

```diff
@@ -1,11 +1,50 @@
 import gql from 'graphql-tag'
 
 export default gql`
+type VaultMetaMigration {
+  available: Boolean!
+  target: String
+  contract: String
+}
+
+type VaultMetaStability {
+  stability: String!
+  stableBaseAsset: String
+}
+
+type VaultMetaInclusion {
+  isYearn: Boolean
+  isGimme: Boolean
+  isPoolTogether: Boolean
+  isCove: Boolean
+  isMorpho: Boolean
+  isKatana: Boolean
+  isPublicERC4626: Boolean
+}
+
 type VaultMeta {
+  name: String
+  registry: String
```

Comment:
`VaultMeta.name` and `VaultMeta.registry` are convenience fields used internally by cms. remove from typedef